docs/faq.rst

```diff
@@ -92,3 +92,22 @@ preview" status. However, you can take a look at `this blog post`_ outlining a
 workable "DIY" solution as a stop gap for now.
 
 .. _`this blog post`: https://www.jeffgeerling.com/blog/2019/how-add-integration-tests-ansible-collection-molecule
+
+Does Molecule support monorepos?
```

Comment:
```suggestion
Does Molecule support "monorepos"?
```

Reply:
After further consideration, I think I'll revert this part since monorepo is actually an established term: https://en.wikipedia.org/wiki/Monorepo (see references also from that article)